### DISCUSSION TITLE
Text overlap in logscale AxisItem

### DISCUSSION THREAD

**Discussion by patoutafay** [pyqtgraph/pyqtgraph#2714] [General]
is it possible to avoid overlap of text in the bottom axis in logscale for example ?
<img width="260" alt="Captur" src="https://user-images.githubusercontent.com/130717926/236106839-78d8cc59-169f-4cee-86d8-7c92d61d1306.png">

**Comment by BarbaRuivaNaMata** [comment]
reduce the amount of ticks on x axis
theres the way i did it

dont forget of 
self.x_axis = plot.getAxis('bottom')

put that on sigrangechanged of viewbox

ViewBox.sigXRangeChanged.connect(updateRangeX) 

def updateRangeX(self,viewRange):
vr0=int(viewRange[0])
        if(vr0 <= 0):
            vr0=1
        vr1=int(viewRange[1])
        total = (int(vr1) - int(vr0))
        ticks_position=[]
        num_ticks = 5 #number of ticks to be expose
        space= total//num_ticks
        x = vr0+space//2
        for pos in self.all_xticks_position:
            if pos >= vr0 and pos < vr1:
                if x == pos:
                    ticks_position.append((pos,self.all_xticks_position[pos][0]))
                    x = x+space
        # print ('TickXPositions:\n',ticks_position)
        self.x_axis.setTicks([ticks_position])   
        
        i am really newbe soo just a ideia dont copy and paste.
